zulip/zulip/__init__.py

```diff
@@ -44,6 +45,31 @@
 
 API_VERSTRING = "v1/"
 
+# An optional parameter to `move_topic` and `update_message` actions
+EditPropagateMode = Literal["change_one", "change_all", "change_later"]
+
+# Generally a `reaction_type` parameter whenever an emoji is specified:
+# - optional parameters to actions: `add_reaction`, `remove_reaction`
+# - events: "user_status", "reaction", "message", "update_message"
+# - generally inside each reaction in the `reactions` field of returned message objects
+EmojiType = Literal["realm_emoji", "unicode_emoji", "zulip_extra_emoji"]
+
+# Personally modifiable message flags:
+# - updated by `update_message_flags`, `mark_all_as_read`, `mark_stream_as_read`, `mark_topic_as_read` actions
+# - notified to the user via "update_message_flag" events
+ModifiableMessageFlag = Literal["read", "starred", "collapsed"]
```

Comment:
This part would actually especially benefit from a link to the API docs:
https://zulip.com/api/update-message-flags#available-flags

because it's otherwise a little bit mysterious what "personally modifiable" means, but the prose in that table makes it clear: these are the flags that can be changed by the user directly in an update-message-flags request.